course_access_groups/utils.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, unicode_literals
+import logging
+from xblock.fields import List, Scope
+from xmodule.course_module import CourseFields
+from django.utils.translation import ugettext as _
+
+log = logging.getLogger(__name__)
+
+
+def add_to_course_fields():
+    """
+    Add the `course_access_groups` field to the course.
+
+    This is monkey patching hence the appropriate warning message.
```

Comment:
Can you describe a bit more on what this does exactly? from a quick scan of the docstring, all I understand is that this is a monkepatch hack for course access groups, but have no idea what it is supposed to do

Reply:
Will reword to make it clearer.

Reply:
Thank you!